pkg/retry/retry.go

```diff
@@ -59,6 +61,9 @@ func isRetryable(err error) bool {
 	case *net.OpError:
 		return isRetryable(e.Err)
 	case *url.Error:
```

Comment:
```suggestion
	case *url.Error: // This includes errors returned by the net/http client.
```
the documentation of just the net/url package does not say this, so it’s not immediately obvious.